src/requests/features.jl

```diff
@@ -376,3 +376,44 @@ function get_module_of(s::StaticLint.Scope, ms = [])
         return isempty(ms) ? "Main" : join(ms, ".")
     end
 end
+
+using Base.Docs, Markdown
+
+function julia_getDocAt_request(params::VersionedTextDocumentPositionParams, server::LanguageServerInstance, conn)
+    uri = URI2(params.textDocument.uri)
+    hasdocument(server, uri) || return nodocuemnt_error(uri)
+
+    doc = getdocument(server, uri)
+    if doc._version !== params.version
+        return mismatched_version_error(uri, doc, params, "getDocAt")
+    end
+
+    x = get_expr1(getcst(doc), get_offset(doc, params.position))
```

Comment:
Shouldn't this be `get_expr`?

Reply:
hm, why do you think so ?
I adapted code from [this line](https://github.com/julia-vscode/LanguageServer.jl/blob/0198aeec2a1310385e431be0e1901f36f3756855/src/requests/hover.jl#L3), and not so sure about the difference between `get_expr` and `get_expr1`...
Maybe @ZacLN could you explain this to me ?